Update claiming docs for Docker containers. (#10570)
netdata/netdata: claim/README.md

### Claim an Agent running in Docker

> The claiming process works with Agents running inside of Docker containers. You can either set a specific set
> of environment variables in the container to have it automatically claimed on startup or restart, or you can use
> `docker exec` to manually claim an already-running container.

Reviewer: ```suggestion
The claiming process works with the Netdat Agent running inside of a Docker container. Use a specific set
of environment variables in the container to have it automatically claimed on startup or restart, or use
`docker exec` to manually claim an already-running container.
```

Author: I’m curious about the reasoning behind shifting to a more imperative wording here.

Reviewer: Lately I've been more critical about Netdata's writing—maybe it's the time on spent on the style guide—and feel we should be more active in our tone. I prefer "do X or Y" over "you can do X or Y" because it's shorter and feels more empowering and inclusive.

We're prolific in our use of "you can," from the beginning of the project. I don't know if I inherited it from the existing docs or brought more of the tendency in myself, but I'm trying to correct myself and move docs away from that style as I approach it. I even wrote a Go script to parse the docs and list every one that uses "you can," among other things...

Reviewer: Also, I typo'd **Netdata** in my suggestion. Whoops.

Author: OK, the shorter part I get, but I personally feel that using `do X or Y` sounds out of place in documentation unless you use it in a way that doesn’t result in it sounding like a command.

IOW, this should probably be ‘To claim an instance of the Netdata Agent running in a Docker container, either do X or do Y’, otherwise it sounds like we’re commanding the reader to do something instead of explaining how to do it. The ‘either/or’ aspect is also kind of important here given how long each of the optional clauses is in the sentence, as it solidifies ahead of time that this is a set of choices.

> Note that for claiming to wokr, the contents of `/var/lib/netdata` _must_ be preserved across container restarts
> using a persistent volume.

Reviewer: Perhaps we can link 

```suggestion
For claiming to work, the contents of `/var/lib/netdata` _must_ be preserved across container restarts
using a persistent volume. See our [recommended `docker run` and Docker Compose examples](/packaging/docker/README.md#create-a-new-netdata-agent-container) for details.
```

#### Using environment variables

The Netdata Docker container looks for the following environment variables on startup:

- `NETDATA_CLAIM_TOKEN`
- `NETDATA_CLAIM_URL`
- `NETDATA_CLAIM_ROOMS`
- `NETDATA_CLAIM_PROXY`

If the token and URL are specified in their corresponding variables _and_ the container is not already claimed, it

> will use these values to attempt to claim the container, automatically adding the node to the specified rooms. If

Reviewer: ```suggestion
will use these values to attempt to claim the container, automatically adding the node to the specified War Rooms. If
```

> a proxy is specified, it will be used for the claiming process and for connecting to the cloud.

Reviewer: ```suggestion
a proxy is specified, it will be used for the claiming process and for connecting to Netdata Cloud.
```

  -v /proc:/host/proc:ro \
  -v /sys:/host/sys:ro \
  -v /etc/os-release:/host/etc/os-release:ro \
  -e NETDATA_CLAIM_TOKEN=TOKEN \

>   -e NETDATA_CLAIM_URL=https://app.netdata.cloud \

Reviewer 2: @joelhans in all cases the claim URL needs to be in double quotes such as "https://app.netdata.cloud" Cc @Ferroin

Author: This was actually my mistake.
